modules/licenses/01_mod.mk

```diff
@@ -14,6 +14,9 @@
 
 ###################### Generate LICENSES files ######################
 
+# _top is the directory containing the top-level Makefile
+_top = $(abspath $(dir $(firstword $(MAKEFILE_LIST))))
```

Comment:
We use lastword in other places: https://github.com/cert-manager/makefile-modules/blob/main/modules/help/01_mod.mk#L16